danndav/bhankas
# #165594881: servercreate account controllers

# What does this PR do?
This PR shows completed routes

# Description of Task to be completed?
add account contoller
add transaction controller
complete test for all routes
work on LFA transaction

# How should this be manually tested?
git fetch and checkout to ft-165594881-servercreate-account-controllers
checkout to the branch
run the application
navigate to /controller folder
click on it to check changes

# Any background context you want to provide?
N/A

# What are the relevant pivotal tracker stories?
#165594881

screenshots (if appropriate)
N/A

Questions:
N/A\
